Extend preflight check in a database restore function (#409)
cloudant/couchbackup: README.md

### `couchrestore`

> * `13`: restore target database is not new and empty.

Reviewer: what about:
```suggestion
* `13`: restore target database must be a new and empty database.
```

Author: This doesn't match the tone of the rest error descriptions though, they are descriptive and not prescriptive, e.g. "10: backup source or restore target database does not exist." not - "10: backup source or restore target database must exist", if you see what I mean.

Reviewer: Yea good point, let's keep what you have then.
